### PR TITLE
Kanakpandit17/search bar

### PR DESCRIPTION
**Pull request by kanakpandit17**
## What does this PR do?
The search bar is made properly visible on the top right corner of the webpage. Also, it has placeholder so that users can understand the purpose of the search bar. The focus-ring is fully visible on clicking.

Solved issue #8693

Type of change: UI Improvement

### CODE REVIEW

**Comment by CarinaWolli** [comment]
@Jaibles do we want to make changed to the search bar? That's how the changes of this PR look: 
Screenshot 2023-05-05 at 15 53 11

**Review by sean-brydon — APPROVED** [review]
Tested and happy with this :) Thanks @kanakpandit17

**Comment by PeerRich** [comment]
CleanShot 2023-05-05 at 22 33 50@2x
the margin-right looks odd

**Review by PeerRich — CHANGES_REQUESTED** [review]
CleanShot 2023-05-05 at 22 33 50@2x
the margin right looks odd

File: apps/web/pages/apps/index.tsx [hunk 1]
```diff
@@ -37,12 +37,13 @@ function AppsSearch({
...
     <TextField
-      className="bg-subtle !border-muted !pl-0 focus:!ring-offset-0"
+    className="bg-subtle !border-muted !pl-15 pr-6 focus:!ring-offset-0 md:w-96"
       addOnLeading={<Search className="text-subtle h-4 w-4" />}
       addOnClassname="!border-muted"
-      containerClassName={classNames("focus:!ring-offset-0", className)}
+      containerClassName={classNames("focus:!ring-offset-0 pr-4 pl-6", className)}
       type="search"
       autoComplete="false"
+      placeholder="Search App Name"
```
Comment (PeerRich) [inline apps/web/pages/apps/index.tsx:46]: we need i18n strings here

File: apps/web/pages/apps/index.tsx [hunk 2]
```diff
@@ -37,12 +37,13 @@ function AppsSearch({
 }) {
   return (
     <TextField
-      className="bg-subtle !border-muted !pl-0 focus:!ring-offset-0"
+    className="bg-subtle !border-muted !pl-15 pr-6 focus:!ring-offset-0 md:w-96"
```
Comment (PeerRich) [inline apps/web/pages/apps/index.tsx:40]: there is a prettier issue here

**Comment by kanakpandit17** [comment]
will make the required changes

**Comment by CarinaWolli** [comment]
@kanakpandit17 we should make the search bar wide enough to show the whole text, right now the text is cut off:
Screenshot 2023-05-08 at 15 13 30

**Comment by Jaibles** [comment]
> @Jaibles do we want to make changed to the search bar? That's how the changes of this PR look: Screenshot 2023-05-05 at 15 53 11

This above is far too big imo. We can't solve every visibility problem by making everything in the app bigger. @kanakpandit17 I'd be happy with any width between `190px` (current width in live) and `220px` (width in Figma) but not bigger. Given that most searches are short and results are instant there's enough space to type and you're likely to see what you need in a few key strokes.

If we want placeholder we can just use "Search..."
CleanShot 2023-05-08 at 14 41 03@2x
.

**Comment by kanakpandit17** [comment]
> This above is far too big IMO and the text is too far away from the magnify icon (it should only be an 8px gap). @kanakpandit17 I'd be happy with any width between `190px` (current width in live) and `220px` ([width in Figma](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A53096&t=dygkprGcVLgpfZib-1)) but not bigger. Given that most searches are short and results are instant there's enough space to type and you're likely to see what you need in a few key strokes. We can't solve every visibility problem by making everything in the app bigger. The search already has a prominent position and we have an app-wide search in the top of the sidebar and accessible through k-bar.

Actually text is completely visible in my search bar, will check out

**Comment by Jaibles** [comment]
@kanakpandit17 Any if this issue on mobile is fixed in your pr?

https://user-images.githubusercontent.com/4536123/236951037-8789b61f-cbe6-4780-b09d-5bce9ab89193.MOV

**Review by Udit-takkar — COMMENTED** [review]
@kanakpandit17 this issue has been fixed by some other contributor. you can check at latest main branch

Screenshot 2023-05-10 at 2 44 53 PM
Screenshot 2023-05-10 at 2 45 16 PM

**Comment by CarinaWolli** [comment]
As @Udit-takkar mentioned, a fix for this already been merged to main (https://github.com/calcom/cal.com/pull/8741)

**Pull request closed without merging**